Source/SwiftLintFramework/Rules/Style/NumberSeparatorRuleExamples.swift

```diff
@@ -13,7 +13,9 @@ internal struct NumberSeparatorRuleExamples {
                 "let hex = \(sign)0xAA_BB",
                 "let octal = \(sign)0o21",
                 "let octal = \(sign)0o21_1",
-                "let exp = \(sign)1_000_000.000_000e2"
+                "let exp = \(sign)1_000_000.000_000e2",
+                "let foo: Double = \(sign)(200)",
+                "let foo: Double = \(sign)(200 / 447.214)"
```

Comment:
can we add a triggering example with parenthesis to document the behavior?

Reply:
Definitely. I'll make a separate PR.